cli/cmd/cluster_gcp.go

```diff
@@ -269,9 +269,21 @@ var _clusterGCPDownCmd = &cobra.Command{
 		if _flagClusterGCPDownKeepVolumes {
 			uninstallCmd += " --keep-volumes"
 		}
-		_, _, err = runGCPManagerAccessCommand(uninstallCmd, *accessConfig, nil, nil)
-		if err != nil {
-			exit.Error(err)
+		output, exitCode, err := runGCPManagerAccessCommand(uninstallCmd, *accessConfig, nil, nil)
+		if (exitCode != nil && *exitCode != 0) || err != nil {
+			var errorMessage string
+			if err != nil {
+				errorMessage = err.Error()
+			} else {
+				errorMessage = output
+			}
+
+			fmt.Print("\n\n")
+			fmt.Print(fmt.Sprintf("￮ failed to uninstall components from cluster: %s", errorMessage))
```

Comment:
`failed to uninstall components from cluster` -> `failed to delete persistent disks from storage, please visit https://console.cloud.google.com/compute/disks?project=%s to manually delete the disks`

Reply:
Did it like this instead: `failed to delete persistent disks from storage, please visit https://console.cloud.google.com/compute/disks?project=%s to manually delete the disks starting with the %s prefix: %s`.